spaces/S000085/properties/P000102.md

```diff
@@ -0,0 +1,15 @@
+---
+space: S000085
+property: P000102
+value: true
+---
+
+Let $X=\mathbb{R}\backslash\{0\} \cup \{0_\alpha: \alpha\in S\}$ be a real line with multiple origins. Define a semimetric $d:X^2\to[0,+\infty)$ by
```

Comment:
```suggestion
Let $X=\mathbb{R}\backslash\{0\} \cup \{0_\alpha: \alpha\in S\}$ be a real line with multiple origins. Define a symmetric $d:X^2\to[0,+\infty)$ by
```
To match the definitions we use in pi-base (but please double check the previous discussions about that), such functions are called "symmetric", independently of any topology.  It is only after the fact, namely after showing it generates the topology with neighborhood bases that we can call it "semimetric" for the topology.